### ISSUE TITLE
Replace "pointDirection" with ATAN2

### ISSUE THREAD

**Issue by norr-carr** [OrionSword/SRVisionProcessing2021#4]
"Also, I have a "pointDirection" function in there from the old code.  I didn't know about the ATAN2 function last year when I wrote that.  ATAN2 is built in and does the same thing." @OrionSword

**Comment by norr-carr** [comment]
Hmm this would make more sense if we used PointDirection right now.  Maybe it should just be removed.

**Comment by CJKeller03** [comment]
Yeah, if PointDirection isn't being used anywhere and atan2 does the same thing, we should just remove it.
